base/shared/src/main/scala/scalaz/data/MaybeSyntax.scala

```diff
@@ -2,5 +2,11 @@ package scalaz
 package data
 
 trait MaybeSyntax {
-  implicit class OptionAsMaybe[A](oa: Option[A]) { def asMaybe: Maybe[A] = Maybe.fromOption(oa) }
+  def empty[A]                     = data.Maybe.empty[A]
+  def just[A](a: A)                = data.Maybe.just(a)
+  def maybe[A, B](n: B)(f: A => B) = data.Maybe.maybe(n)(f)
+  def fromOption[A](oa: Option[A]) = data.Maybe.fromOption(oa)
+  def toOption[A](ma: Maybe[A])    = data.Maybe.toOption(ma)
```

Comment:
Again, why are these syntax? I think we need to clarify what counts as "syntax".

Reply:
Same as above, putting them back